RecoLocalCalo/EcalRecProducers/plugins/EcalPh2DigiToGPUProducer.cc

```diff
@@ -0,0 +1,105 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "HeterogeneousCore/CUDACore/interface/ScopedContext.h"
+#include "CUDADataFormats/EcalDigi/interface/DigisCollection.h"
+
+#include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
+#include "DataFormats/EcalDigi/interface/EcalDataFrame_Ph2.h"
+
+#include "DeclsForKernelsPh2.h"
+
+class EcalPh2DigiToGPUProducer : public edm::stream::EDProducer<edm::ExternalWork> {
```

Comment:
Please write out `Phase2` in the types, functions and file names:
```suggestion
class EcalPhase2DigiToGPUProducer : public edm::stream::EDProducer<edm::ExternalWork> {
```